EventFilter/Phase2TrackerRawToDigi/plugins/Phase2TrackerCommissioningDigiProducer.cc

```diff
@@ -48,7 +48,7 @@ void Phase2Tracker::Phase2TrackerCommissioningDigiProducer::produce(edm::StreamI
 
   // Analyze strip tracker FED buffers in data
   size_t fedIndex;
-  for (fedIndex = 0; fedIndex < Phase2Tracker::CMS_FED_ID_MAX; ++fedIndex) {
+  for (fedIndex = 0; fedIndex <= Phase2Tracker::CMS_FED_ID_MAX; ++fedIndex) {
```

Comment:
`Phase2Tracker::CMS_FED_ID_MAX` is equal to `FEDNumbering::MAXFEDID`:
https://github.com/cms-sw/cmssw/blob/038eb56268f2fb41e45e46828e556c174dc1f589/EventFilter/Phase2TrackerRawToDigi/interface/utils.h#L19